src/components/Infobar/Infobar.tsx

```diff
@@ -31,14 +30,17 @@ export const InfoBar = () => {
       {state.startTime && state.endTime && <Timer startTime={state.startTime} endTime={state.endTime} />}
       {state.activeVoting && <VoteDisplay usedVotes={state.usedVotes} possibleVotes={state.possibleVotes!} />}
       {state.sharedNote && viewer.user.id !== focusInitiator?.user.id && (
-        <TooltipButton
-          className="info-bar__return-to-presented-note-button"
-          icon={ShareIcon}
-          direction="right"
-          label={t("InfoBar.ReturnToPresentedNote")}
-          onClick={() => navigate(`note/${state.sharedNote}/stack`)}
-        />
+        <Link
+          aria-label={t("InfoBar.ReturnToPresentedNote")}
+          className="info-bar__return-to-shared-note-button"
+          data-tooltip-id="info-bar__tooltip"
```

Comment:
I guess this id must be referencing the id of the `Tooltip` component.
If not, it may be sensible to use a more meaningful id which references the use of the shared note button instead of a general tooltip.

Reply:
It refers to the tooltip used for the info bar